tomee/tomee-microprofile/mp-common/pom.xml

```diff
@@ -138,7 +260,7 @@
     <dependency>
       <groupId>org.eclipse.microprofile.metrics</groupId>
       <artifactId>microprofile-metrics-api</artifactId>
-      <version>${microprofile.health.version}</version>
+      <version>${microprofile.metrics.version}</version>
```

Comment:
Good catch but if you have dependency management, shouldn't we take the version out?

Reply:
took out all the managed versions